### PR TITLE
Printing linker error message from  gradle MPP plugin

### PR DESCRIPTION
**Pull request by LepilkinaElena**

### CODE REVIEW

File: shared/src/main/kotlin/org/jetbrains/kotlin/konan/exec/ExecuteCommand.kt [hunk 1]
```diff
@@ -47,23 +48,23 @@ open class Command(initialCommand: List<String>) {
...
+    open fun runProcess() {
         val builder = ProcessBuilder(command)
 
         builder.redirectOutput(Redirect.INHERIT)
         builder.redirectInput(Redirect.INHERIT)
-        builder.redirectError(Redirect.INHERIT)
 
         val process = builder.start()
+
+        val reader = BufferedReader(InputStreamReader(process.errorStream))
```
Comment (PavelPunegov) [inline shared/src/main/kotlin/org/jetbrains/kotlin/konan/exec/ExecuteCommand.kt]: It seems that error stream is now will not be visible if process exits correctly. Where are all warnings printed? Into err or std out?

Comment (LepilkinaElena) [reply shared/src/main/kotlin/org/jetbrains/kotlin/konan/exec/ExecuteCommand.kt]: Before nothing is seen if run with gradle plugin, only if run `konanc` you got all messages. I think how to save both modes.

Comment (LepilkinaElena) [reply shared/src/main/kotlin/org/jetbrains/kotlin/konan/exec/ExecuteCommand.kt]: Warnings are printed in stderr

File: shared/src/main/kotlin/org/jetbrains/kotlin/konan/exec/ExecuteCommand.kt [hunk 2]
```diff
@@ -47,23 +48,23 @@ open class Command(initialCommand: List<String>) {
...
+
         val exitCode = process.waitFor()
-        return exitCode
+        handleExitCode(exitCode, reader.readLines())
```
Comment (PavelPunegov) [inline shared/src/main/kotlin/org/jetbrains/kotlin/konan/exec/ExecuteCommand.kt]: It doesn't return anything, while it's required in Linker.kt

Comment (PavelPunegov) [reply shared/src/main/kotlin/org/jetbrains/kotlin/konan/exec/ExecuteCommand.kt]: See https://buildserver.labs.intellij.net/viewLog.html?buildId=66946137&problemId=1583664

**Pull request merged**